content/docs-v0/pagedjs.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Creating PDFs with Paged.js
+weight: 5700
+type: essay
+menu: false
+abstract: "Using Paged Media CSS and Paged.js to create PDF output"
+---
+
+Paged.js is an open source JavaScript library that implements the Paged Media standards published by the W3C (ie the Paged Media Module, and the Generated Content for Paged Media Module) to create PDF output from any HTML content from within the browser.
+
+Paged.js acts as a polyfill for the CSS modules enabling CSS features that are not yet natively supported by browsers, while leveraging the parts that have been implemented.
+
+Documentation and examples of CSS Paged Media can be found on [pagedjs.org](https://pagedjs.org/documentation/)
+
+[Paged.js — Web design for print](https://pagedjs.org/documentation/5-web-design-for-print/) is a great place to get started if you are new to print css.
+
+## Compatibility
+Not every part of the Paged Media specification has been implemented yet, but it continues to be worked on at [gitlab.coko.foundation/pagedjs/pagedjs](https://gitlab.coko.foundation/pagedjs/pagedjs/).
+
+If a feature is missing or not working, please file an issue there so it can be triaged and tested.
+
+However there are a few features to call out as unlikely to be supported anytime soon:
+- [ ] CMYK output
+- [ ] CSS grids that are larger than a single page
+- [ ] PDF/A profile output
+- [ ] Any `-prince`  prefixed CSS rules
+- [ ] Dot leaders
+
+Paged.js strives to have similar output as other Paged Media PDF generators and is mostly consistent with the output from Prince XML. Print CSS Rocks maintains an excellent comparison of the results of using the same CSS with different generators at [print-css.rocks/lessons](https://print-css.rocks/lessons)
+
+## Quirks
+There are few quirks to be aware of when using Paged.js, that hopefully should be fixed as soon as possible.
+
+### Page Size
+Due to limitations with cross-browser PDF generation,  only the initial `size` declaration will be used.
+
+```css
+// USED
+@page {
+	size: 6in 10in;
+}
+// Ignored
+@page cover {
+	size: 10in 20in;
+}
+```
+
+### @page Rules
+Due to CSS parsing issues, all @page rules should be separate entries.
+
+```css
+// WORKS
+@page {
+	size: A5
+	@bottom-left {
+  		content: counter(page);
+	}
+}
+
+@page cover {
+	background-color: purple;
+	@bottom-left {
+  		content: counter(page);
+	}
+}
+
+// FAILS
+@page {
+	size: A5
+}
+
+@page cover {
+	background-color: purple;
+}
+
+@page, @page cover {
+	@bottom-left {
+  		content: counter(page);
+	}
+}
+```
```

Comment:
It may make the example more clear to separate and order the examples *fails* and *works*, thoughts?

```suggestion
*FAILS*
```
@page {
	size: A5
}

@page cover {
	background-color: purple;
}

@page, @page cover {
	@bottom-left {
  		content: counter(page);
	}
}
```

*WORKS*
```css
@page {
	size: A5
	@bottom-left {
  		content: counter(page);
	}
}

@page cover {
	background-color: purple;
	@bottom-left {
  		content: counter(page);
	}
}
```
```